examples/http-failover/http_failover.client.out

```diff
@@ -3,7 +3,15 @@ $ curl -v http://localhost:9090/fo
 # The `FailoverClient` endpoint is configured with three target services.The first two targets are configured to mimic
 # failure backends. If one target service goes down, the failover client automatically calls the other targets.
 # Once you invoke the Failover demo service, the Failover client fails over the request to the configured
-# target endpoints. In this example, the third target is configured to give a successful request and the following 
+# target endpoints. In this example, the third target is configured to give a successful response and the following
 # response is given when the failover demo service is invoked.
 # Server response:
 Mock Resource is Invoked.
+# Invoke the Failover Service again using below curl command.
```

Comment:
In ballerina.io, the comments are rendered on the side of the command so we cannot say 'below'. Just say 'using the curl command.'